### PR TITLE
Add cellery setup command - initial changes

### PR DESCRIPTION
**Pull request by MadhukaHarith92**
Following are the selections of cellery setup 
1. manage - start, stop VM 
2. create - local environment 
3. switch

### CODE REVIEW

File: components/cli/pkg/util/utils.go [hunk 1]
```diff
@@ -514,3 +522,86 @@ func GetFileSize(path string) (int64, error) {
 
 	return file.Size(), nil
 }
+
+func RenameFile(oldName, newName string) error {
```
Comment (anuruddhal) [inline components/cli/pkg/util/utils.go]: The function name should start with a simple letter.

File: components/cli/pkg/util/utils.go [hunk 2]
```diff
@@ -514,3 +522,86 @@ func GetFileSize(path string) (int64, error) {
...
+	err := os.Rename(oldName, newName)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func ReplaceFile(fileToBeReplaced, fileToReplace string) error {
```
Comment (anuruddhal) [inline components/cli/pkg/util/utils.go]: The function name should start with a simple letter. Please fix the below functions as well.

File: components/cli/pkg/commands/setup.go [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+	"github.com/celleryio/sdk/components/cli/pkg/util"
+	"github.com/fatih/color"
+	"github.com/manifoldco/promptui"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+func RunSetup() error {
```
Comment (anuruddhal) [inline components/cli/pkg/commands/setup.go]: The function name should start with a simple letter.

File: components/cli/pkg/commands/setup.go [hunk 4]
```diff
@@ -0,0 +1,299 @@
...
+			fmt.Println("Installing GCP")
+		}
+		default: {
+			RunSetup()
+		}
+	}
+
+	fmt.Printf(util.GreenBold("\n\U00002714") + " Successfully installed Cellery runtime.\n")
+	fmt.Println()
+	fmt.Println(bold("Whats next ?"))
```
Comment (anuruddhal) [inline components/cli/pkg/commands/setup.go]: Should be "What's"

File: components/cli/pkg/commands/setup.go [hunk 5]
```diff
@@ -0,0 +1,299 @@
...
+	}
+	_, value, err := cellPrompt.Run()
+	if err != nil {
+		return fmt.Errorf("failed to select cluster: %v", err)
+	}
+
+	setContext(value)
+	fmt.Printf(util.GreenBold("\n\U00002714") + " Successfully configured Cellery.\n")
+	fmt.Println()
+	fmt.Println(bold("Whats next ?"))
```
Comment (anuruddhal) [inline components/cli/pkg/commands/setup.go]: Should be "What's"

Comment (anuruddhal) [inline components/cli/pkg/commands/setup.go]: Should be "What's"

File: components/cli/pkg/util/utils.go (hunk 1 above)
Comment (MadhukaHarith92) [reply components/cli/pkg/util/utils.go]: Discussed.

File: components/cli/pkg/commands/setup.go (hunk 4 above)
Comment (MadhukaHarith92) [reply components/cli/pkg/commands/setup.go]: Done

File: components/cli/pkg/commands/setup.go (hunk 5 above)
Comment (MadhukaHarith92) [reply components/cli/pkg/commands/setup.go]: Done

**Pull request merged**